RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -994,11 +995,42 @@ namespace SDL {
     }
   };
 
+  // Helper function to determine eta bin for occupancies
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE int getEtaBin(const float module_eta) {
+    if (module_eta < 0.75f)
+      return 0;
+    if (module_eta < 1.5f)
```

Comment:
```suggestion
    else if (module_eta < 1.5f)
```
since we tend to get these in the PR reviews

Reply:
Fixed in latest commit.